# Pregunta 9
## ¿Del total de carros que porcentaje tiene menos de k años?

In [2]:
import numpy as np
import pandas as pd
from scipy import stats, integrate
import matplotlib.pyplot as plt
import seaborn as sns
from __future__ import division 
df = pd.read_csv('allstate.csv')
#Declaro funcion recibe de parametro los años que queremos que sea menor
def CDF( k ):
    filtrok = df.car_age<=k
    MenorQ=df.car_age[(filtrok)&(df.record_type==1)].count()
    TotalCarr=df.car_age[df.record_type==1].count()
    Probabilidad = MenorQ/TotalCarr
    return str(Probabilidad*100)+'%'
CDF(0)

'0.918471482027%'

## Respuesta: 
## Para k=0 hay 0.918% carros
## Para k=1 hay 11.6143862941% carros
## Para k=5 hay 36.1842715624% carros
## Para k=10 hay 69.0111226793% carros
## Para k=20 hay 97.2332463998% carros
## Para k=30 hay 99.6453937264% carros
## Para k=85 hay 100.00% carros